developer_manual/scripts/theme-bootstrap.sh

```diff
@@ -14,44 +15,90 @@ then
   exit $E_BADARGS
 fi
 
+# Utility function for installing missing packages
+# This only works on Debian-based distros or those that support Apt
+function install_package
```

Comment:
Maybe I am blind but
what is ``function install_package`` and ``function install_required_binaries`` good for when cloning an app from github?
It uses btw the same parameters ``$1`` which is the app-name from owncloud/git and a package you usually get from apt?

Reply:
The intent of these two functions is to make sure that the required dependencies have been installed and not to fail if they haven't. Ah, I see an error in my thinking. Those packages would be run by an admin and the cloning/setup steps would be run by the web server user. Will correct.